runtime/interface.cpp

```diff
@@ -1544,8 +1545,8 @@ static void save_rpc_query_headers(const tl_query_header_t &header) {
   }
 }
 
-static void init_superglobals(const http_query_data &http_data, const rpc_query_data &rpc_data, const job_query_data &job_data) {
-  rpc_parse(rpc_data.data, rpc_data.len);
+static void init_superglobals_impl(const http_query_data &http_data, const rpc_query_data &rpc_data, const job_query_data &job_data) {
+  rpc_parse(rpc_data.data.data(), rpc_data.data.size());
```

Comment:
now here's a bug: `rpc_data.data.size()` is one int-size larger than before
I think we need to get rid of that strange `+1` in `rpcx_execute` and all will be fine